R/servers.R

```diff
@@ -97,5 +90,14 @@ server_urls <- c(
   "http://data.whiterockcity.ca/",
   "https://catalogue.data.gov.bc.ca",
   "https://data.montreal.ca",
-  "https://www.donneesquebec.ca"
+  "https://www.donneesquebec.ca",
+  "http://dados.tce.rs.gov.br/",
+  "https://dados.al.gov.br/catalogo/",
+  "https://dati.comune.roma.it/catalog/",
+  "https://www.dati.gov.it/",
+  "https://data.amerigeoss.org/",
+  "https://opendata-ajuntament.barcelona.cat/",
```

Comment:
this one returns a 404 not found as is - if it requires `/data`, then add that to that url 

also appears to have a misconfigured SSL cert